service/AppData.py

```diff
@@ -149,3 +153,21 @@ def get_peer_file_by_index(cls, index: int) -> tuple:
 	def clear_peer_files(cls) -> None:
 		cls.peer_files.clear()
 	# -----------------------------------------------------------------------------
+
+	# query management-------------------------------------------------------------
```

Comment:
Io terrei in memoria solo il pktid, tanto è l'unica cosa che ci interessa realmente controllare